### ISSUE TITLE
[Question] Why does the group 千 of 剰 only consists of one part

### ISSUE THREAD

**Issue by CaptainDario** [KanjiVG/kanjivg#423]
<a href="https://github.com/KanjiVG/kanjivg/blob/master/kanji/05270.svg"><img src="https://raw.githubusercontent.com/KanjiVG/kanjivg/master/kanji/05270.svg" width="150" height="150"></a>

This kanji has two 'part' definitions with a "1" as value.

``` svg
<g id="kvg:05270-Kaisho-g2" stroke="red" kvg:element="千" kvg:part="1">
<g id="kvg:05270-Kaisho-g4" kvg:element="十" kvg:part="1">
```

However, only 十 has a `kvg:part=2` definition. Is this intended? Could you explain why `id="kvg:05270-Kaisho-g6"` does not have a `kvg:part="2"`. For me this would be necessary as it defines the second part of this group

``` svg
<g id="kvg:05270-Kaisho-g6" kvg:element="千">
  <g id="kvg:05270-Kaisho-g7" kvg:element="十" kvg:part="2">
    <path id="kvg:05270-Kaisho-s7" kvg:type="㇑" d="M38.82,21.38c0.65,0.45,1.29,2.12,1.29,4.25c0,5.37-0.21,64.64-0.21,65.25c0,10.62-5.15,2.37-6.4,1.12"/>
  </g>
</g>
```

**Comment by benkasminbullock** [comment]
Thank you for reporting this omission. It should be fixed by 8048febd4. Please reopen this issue if you are not satisfied. 05270-Kaisho.svg seems to have the missing kvg:part="2" already.

To answer your question about "why", there are innumerable errors in the KanjiVG data regarding the parts and groups and stroke types. The original data was created by hand by a group of university students I believe. I've been trying to sort this out but it's ongoing work. Please open an issue when you see something like this.

**Comment by CaptainDario** [comment]
Thank you!

I am using KanjiVG for my Japanese dictionary app. Last month I published it and am using it myself. While using it I am seeing sometimes weird results, and as I am not sure if the problem is my data processing or the data itself I rather assume I made a mistake.
Anyways, thanks again and I guess I will open more issues in the future.
